presto-hive/src/main/java/io/prestosql/plugin/hive/RecordFileWriter.java

```diff
@@ -68,6 +68,7 @@
     private final long estimatedWriterSystemMemoryUsage;
 
     private boolean committed;
+    private long committedPathSize;
```

Comment:
Let's call this `finalWrittenBytes` to indicate that this is the final size (won't change)